resources/public/dashboard/index.html

```diff
@@ -91,6 +91,9 @@
 
 <script src="d3.v5.js"></script>
 <script src="charts.js"></script>
+<!-- token needed here ? -->
+<!-- <script src="d3.v5.js?token="></script>
+<script src="charts.js?token="></script> -->
```

Comment:
Just want to make sure we remember to remove these before merging.

Reply:
got catch. I forgot about them 👍